docs/source/spec/aws-core.rst

```diff
@@ -592,6 +592,107 @@ previous example:
         }
 
 
+.. _aws.api#data-trait:
+
+``aws.api#data`` trait
```

Comment:
This should call out that data classifications are scoped. If you apply the trait to a structure, then all members inherit the data classification of the structure unless they override it. I don't recall if this property holds true in more high-level ways like applying it to an operation or service though.